repository/src/main/java/org/apache/atlas/repository/store/graph/v2/utils/TagAttributeMapper.java

```diff
@@ -0,0 +1,352 @@
+package org.apache.atlas.repository.store.graph.v2.utils;
+
+import org.apache.atlas.AtlasErrorCode;
+import org.apache.atlas.exception.AtlasBaseException;
+import org.apache.atlas.model.TypeCategory;
+import org.apache.atlas.model.instance.AtlasClassification;
+import org.apache.atlas.model.typedef.AtlasStructDef.AtlasAttributeDef;
+import org.apache.atlas.type.AtlasArrayType;
+import org.apache.atlas.type.AtlasClassificationType;
+import org.apache.atlas.type.AtlasEntityType;
+import org.apache.atlas.type.AtlasMapType;
+import org.apache.atlas.type.AtlasStructType;
+import org.apache.atlas.type.AtlasStructType.AtlasAttribute;
+import org.apache.atlas.type.AtlasType;
+import org.apache.atlas.type.AtlasTypeRegistry;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.collections.MapUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Utility class for mapping classification attributes in a consistent format for v1 and v2.
+ * Used to ensure that classifications stored in Cassandra via v2 flow maintain the same
+ * structure as classifications stored in JanusGraph via v1 flow.
+ */
+@Component
```

Comment:
As I suggested earlier, remove Component & have methods as static

Reply:
This class needs an instance of typeRegistry which is a bean. So if we keep it static, we won't be able to inject typeRegistry which then needs to be passed as a method argument.

The caller classes would need to add dependency on typeRegistry and pass it so hence this code is simpler and avoids lot of changes in different client classes.